[OSDOCS#16946]: Make changes as per the errors and warnings for CQA-1 (#114341)
openshift/openshift-docs: modules/storage-persistent-storage-volume-encrypt-with-kms-key.adoc

[id="aws-container-persistent-volumes-encrypt_{context}"]
= Encrypting container persistent volumes on AWS with a KMS key

[role="_abstract"]

> Defining a KMS key to encrypt container-persistent volumes on AWS is useful when you have explicit compliance and security guidelines when deploying to AWS.

Reviewer: ```suggestion
[role="_abstract"]
You can define a KMS key to encrypt container-persistent volumes on AWS if you have explicit compliance and security guidelines when deploying to AWS.
```

[id="aws-container-persistent-volumes-encrypt_{context}"]
= Encrypting container persistent volumes on AWS with a KMS key

[role="_abstract"]

> You can define a KMS key to encrypt container-persistent volumes on AWS is useful when you have explicit compliance and security guidelines when deploying to AWS.

Reviewer: ```suggestion
You can define a KMS key to encrypt container-persistent volumes on AWS if you have explicit compliance and security guidelines when deploying to AWS.
```
